google/cloud/bigtable/examples/bigtable_examples_common.cc

```diff
@@ -22,24 +22,19 @@ namespace examples {
 
 Example::Example(std::map<std::string, CommandType> commands)
     : commands_(std::move(commands)) {
-  std::string full_usage;
+  full_usage_.clear();
```

Comment:
Typically we would choose not to do this redundant step in the constructor, I think.

Reply:
Will fix too.